packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -90,6 +90,14 @@ export const PARTITION_MAP: { [region: string]: Region } = {
   'us-isob-': { partition: Partition.UsIsoB, domainSuffix: 'sc2s.sgov.gov' },
 };
 
+export const CR_DEFAULT_RUNTIME_MAP: Record<Partition, string> = {
```

Comment:
This needs be the value that feeds into `Runtime.NODEJS_LATEST`. There is no way to do `NODEJS_LATEST` correctly with a hardcoded constant.